agents/sorenson_dlm/dlm_agent.py

```diff
@@ -255,6 +255,49 @@ def set_current(self, session, params=None):
 
         return True, 'Set current to {}'.format(params['current'])
 
+    @ocs_agent.param('auto_acquire', default=False, type=bool)
+    @ocs_agent.param('acq_params', type=dict, default=None)
+    @ocs_agent.param('force', default=False, type=bool)
+    def init_dlm(self, session, params=None):
+        """init_dlm(auto_acquire=False, acq_params=None, force=False)
+        **Task** - Perform first time setup of the Sorenson DLM communication.
+
+        Parameters:
+            auto_acquire (bool, optional): Default is False. Starts data
+                acquisition after initialization if True.
+            acq_params (dict, optional): Params to pass to acq process if
+                auto_acquire is True.
+            force (bool, optional): Force initialization, even if already
+                initialized. Defaults to False.
+        """
+        if params is None:
+            params = {}
```

Comment:
This is no longer needed, as you can assume the `params` dict exists since you're using the param decorator.